ocaml/xapi/xapi_globs.ml

```diff
@@ -1194,6 +1216,9 @@ module Resources = struct
     ; ( "generate_ssl_cert"
       , Constants.generate_ssl_cert
       , "script to generate SSL certificates to be used by XAPI" )
+    ; ( "gen_pool_secret_script"
+      , gen_pool_secret_script
+      , "Generates new pool secrets" )
```

Comment:
Even if urandom is used, the script will need to be in the filesystem. Not sure if it should be required. Is there another solution for this?
In any case @stormi could work around this by placing an empty file.

Reply:
As discussed we'll default to using /dev/urandom, and only use the script if the file /usr/bin/pool_secret_wrapper exists